src/renderer/atlas/AtlasEngine.cpp

```diff
@@ -315,6 +315,15 @@ CATCH_RETURN()
         }
 
         _api.selectionSpans = til::point_span_subspan_within_rect(info.selectionSpans, dr);
+
+        u32 newSelectionColor{ static_cast<COLORREF>(info.selectionBackground) | 0xff000000 };

```

Comment:
@lhecker please check me on this - I don't think there's another place for us to propagate this...

Reply:
Seems good to me!